test/parallel/test-http-request-agent.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: I don't think this should be moved either. I think it should be left where it was.